scripts/patchwheel.py

```diff
@@ -0,0 +1,40 @@
+import shutil,subprocess,argparse
+from pathlib import Path
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Patch Arbor wheels build with skbuild and correct by auditwheel. Linux only.')
+    parser.add_argument('path', type=dir_path, help='The path in which the wheels will be patched.')
```

Comment:
Is this the path of the old wheels? Or the path of where the new wheels will go? Or both?